From 1e4c3016547b5f75e515d4b9a7202733c47f5a18 Mon Sep 17 00:00:00 2001
From: Andrew Davis <afd@ti.com>
Date: Wed, 4 Dec 2024 14:46:17 -0600
Subject: [PATCH] fix: Fix indention in cmake generator script

All other items in these lists use tabs, the first couple
lines should match. Switch them to tabs then regenerate
the CMakeLists.txt file.

While here fix the missing ending newline.

Upstream-Status: Pending

Signed-off-by: Andrew Davis <afd@ti.com>
Change-Id: Icab852fc43ae08e7858857881c996d233c835289
---
 scripts/generate_build_files.py | 27 ++++++++++++++-------------
 src/CMakeLists.txt              | 26 +++++++++++++-------------
 2 files changed, 27 insertions(+), 26 deletions(-)

diff --git a/scripts/generate_build_files.py b/scripts/generate_build_files.py
index f88cf1af44..52a8cc14da 100644
--- a/scripts/generate_build_files.py
+++ b/scripts/generate_build_files.py
@@ -168,28 +168,29 @@ def build_from_template_cmake(srcs_graph, srcs_sve, srcs_sve2, srcs_core):
     template = f"""{get_template_header()}

 target_sources(
-    arm_compute_graph
-    PRIVATE
-    {line_separator.join(srcs_graph)}
+	arm_compute_graph
+	PRIVATE
+	{line_separator.join(srcs_graph)}
 )

 target_sources(
-    arm_compute_sve
-    PRIVATE
-    {line_separator.join(srcs_sve)}
+	arm_compute_sve
+	PRIVATE
+	{line_separator.join(srcs_sve)}
 )

 target_sources(
-    arm_compute_sve2
-    PRIVATE
-    {line_separator.join(srcs_sve2)}
+	arm_compute_sve2
+	PRIVATE
+	{line_separator.join(srcs_sve2)}
 )

 target_sources(
-    arm_compute
-    PRIVATE
-    {line_separator.join(srcs_core)}
-)"""
+	arm_compute
+	PRIVATE
+	{line_separator.join(srcs_core)}
+)
+"""
     return template


diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index ff3bed96df..f1b275532a 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -21,9 +21,9 @@
 # SOFTWARE.

 target_sources(
-    arm_compute_graph
-    PRIVATE
-    graph/DataLayerVisitor.cpp
+	arm_compute_graph
+	PRIVATE
+	graph/DataLayerVisitor.cpp
	graph/Graph.cpp
	graph/GraphBuilder.cpp
	graph/GraphContext.cpp
@@ -101,9 +101,9 @@ target_sources(
 )

 target_sources(
-    arm_compute_sve
-    PRIVATE
-    core/NEON/kernels/arm_conv/depthwise/interleaves/sve_s8q_3x3_dot.cpp
+	arm_compute_sve
+	PRIVATE
+	core/NEON/kernels/arm_conv/depthwise/interleaves/sve_s8q_3x3_dot.cpp
	core/NEON/kernels/arm_conv/depthwise/interleaves/sve_u8q_3x3_dot.cpp
	core/NEON/kernels/arm_conv/depthwise/kernels/sme2_fp16_nhwc_3x3_s1_output2x2_mla_depthfirst/generic_direct.cpp
	core/NEON/kernels/arm_conv/depthwise/kernels/sme2_fp16_nhwc_3x3_s1_output2x2_mla_depthfirst/generic_indirect.cpp
@@ -332,9 +332,9 @@ target_sources(
 )

 target_sources(
-    arm_compute_sve2
-    PRIVATE
-    cpu/kernels/activation/generic/sve2/lut.cpp
+	arm_compute_sve2
+	PRIVATE
+	cpu/kernels/activation/generic/sve2/lut.cpp
	cpu/kernels/activation/generic/sve2/qasymm8.cpp
	cpu/kernels/activation/generic/sve2/qasymm8_signed.cpp
	cpu/kernels/activation/generic/sve2/qsymm16.cpp
@@ -357,9 +357,9 @@ target_sources(
 )

 target_sources(
-    arm_compute
-    PRIVATE
-    c/AclContext.cpp
+	arm_compute
+	PRIVATE
+	c/AclContext.cpp
	c/AclOperator.cpp
	c/AclQueue.cpp
	c/AclTensor.cpp
@@ -1049,4 +1049,4 @@ target_sources(
	runtime/experimental/operators/CpuSub.cpp
	runtime/experimental/operators/CpuTranspose.cpp
	runtime/experimental/operators/CpuWinogradConv2d.cpp
-)
\ No newline at end of file
+)
